### PR TITLE
feat: Add User Section with JSON Map file

### PR DESCRIPTION
**Pull request by TheShubham99**
- [x] User section with user message having dynamic responses
fetched from the JSON file.
- [x] Scrollbar for a longer message in gooey/user messages (Responsive)
- [x] Placeholder User Image and Username.
- [x] fix: Chat arrow not visible due to `overflow: auto`
- [x] style: minor restyling in the overall Landing Page and Color Scheme
- [x] style: Add formatting to index.css

### CODE REVIEW

**Comment by TheShubham99** [comment]
@pollend @majordwarf 
Kindly have a look 👍

File: src/components/BottomNavbar.jsx [hunk 1]
```diff
@@ -38,12 +41,18 @@ export function BottomNavbar() {
       setpostfix(Constants.readMore);
     }
     setisCollapsed(!isCollapsed);
+
+    //only for mobile phones (handling the overflowed chat arrow element `gooey-message:after`)
+    if (window.outerWidth <= 600) {
+      gooeyMessage.current.style.overflow =
+        gooeyMessage.current.style.overflow === 'auto' ? 'visible' : 'auto';
```
Comment (pollend) [inline src/components/BottomNavbar.jsx:48]: I think you can accomplish this with css.

https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

Comment (TheShubham99) [reply src/components/BottomNavbar.jsx:48]: @pollend Thanks for the review :) 👍


I have used the media queries, but this style/dom modification is based on a button click.

I.e. The styling needs to be changed on button click (read more button)

Intention - to set the overflow to auto if the message is longer (on button click) 

Again set the overflow to visible after clicking on see less.

Reason - We've used :after property for the message bubble `arrow` in gooey message. So if we set overflow:auto (which makes the message scrollable) then :after gets invisible (due to overflow behaviour)

Comment (pollend) [reply src/components/BottomNavbar.jsx:48]: you can append a class conditionally with react by passing it to a prop. have a boolean flag and some condition for the class to be used.

Comment (TheShubham99) [reply src/components/BottomNavbar.jsx:48]: Ohk @pollend will give it a try :) thanks btw :)

Comment (TheShubham99) [reply src/components/BottomNavbar.jsx]: @pollend Done. Kindly have a look.

**Comment by TheShubham99** [comment]
Thanks for the merge @pollend :)

**Pull request merged**